From b791691491c14a7698509ad80406f0a3c9bad298 Mon Sep 17 00:00:00 2001
From: Ludovic Desroches <ludovic.desroches@atmel.com>
Date: Thu, 3 Jan 2013 17:30:30 +0100
Subject: [PATCH 275/319] SAMA5: remove all mb-revB device tree part

Remove mb-revB dts files since dtb is modified on the fly by the bootloader.

Signed-off-by: Ludovic Desroches <ludovic.desroches@atmel.com>
---
 arch/arm/boot/dts/sama5d3.dtsi               |    9 -
 arch/arm/boot/dts/sama5d31ek_mb-revB.dts     |  196 ----------------------
 arch/arm/boot/dts/sama5d31ek_mb-revB_pda.dts |  218 ------------------------
 arch/arm/boot/dts/sama5d33ek_mb-revB.dts     |  196 ----------------------
 arch/arm/boot/dts/sama5d33ek_mb-revB_pda.dts |  218 ------------------------
 arch/arm/boot/dts/sama5d34ek_mb-revB.dts     |  206 -----------------------
 arch/arm/boot/dts/sama5d34ek_mb-revB_pda.dts |  228 --------------------------
 arch/arm/boot/dts/sama5d35ek_mb-revB.dts     |  199 ----------------------
 8 files changed, 1470 deletions(-)
 delete mode 100644 arch/arm/boot/dts/sama5d31ek_mb-revB.dts
 delete mode 100644 arch/arm/boot/dts/sama5d31ek_mb-revB_pda.dts
 delete mode 100644 arch/arm/boot/dts/sama5d33ek_mb-revB.dts
 delete mode 100644 arch/arm/boot/dts/sama5d33ek_mb-revB_pda.dts
 delete mode 100644 arch/arm/boot/dts/sama5d34ek_mb-revB.dts
 delete mode 100644 arch/arm/boot/dts/sama5d34ek_mb-revB_pda.dts
 delete mode 100644 arch/arm/boot/dts/sama5d35ek_mb-revB.dts

diff --git a/arch/arm/boot/dts/sama5d3.dtsi b/arch/arm/boot/dts/sama5d3.dtsi
index 674e7c1..3b53365 100644
--- a/arch/arm/boot/dts/sama5d3.dtsi
+++ b/arch/arm/boot/dts/sama5d3.dtsi
@@ -595,19 +595,10 @@
 							 2 29 0x3 0x0	/* PC29 periph C ISI_PD8 */
 							 2 28 0x3 0x0>;	/* PC28 periph C ISI_PD9 */
 					};
-					pinctrl_isi_pck_as_mck_mb_revB: isi_pck_as_mck_mb_revB-0 {
-						atmel,pins =
-							<2 15 0x2 0x0>;	/* PC15 periph B ISI_MCK */
-					};
 					pinctrl_isi_pck_as_mck: isi_pck_as_mck-0 {
 						atmel,pins =
 							<3 31 0x2 0x0>;	/* PD31 periph B ISI_MCK */
 					};
-					pinctrl_isi_camera_mb_revB: isi_camera_mb_revB-0 {
-						atmel,pins =
-							<4 28 0x0 0x0	/* PE28 gpio ZB_SLPTR */
-							 4 29 0x0 0x0>;	/* PE29 gpio ZB_RSTN */
-					};
 					pinctrl_isi_camera: isi_camera-0 {
 						atmel,pins =
 							<4 24 0x0 0x0	/* PE24 gpio ZB_SLPTR */
diff --git a/arch/arm/boot/dts/sama5d31ek_mb-revB.dts b/arch/arm/boot/dts/sama5d31ek_mb-revB.dts
deleted file mode 100644
index 17d8151..0000000
--- a/arch/arm/boot/dts/sama5d31ek_mb-revB.dts
+++ /dev/null
@@ -1,196 +0,0 @@
-/*
- * sama5d31ek.dts - Device Tree file for SAMA5-EK board
- *
- *  Copyright (C) 2012 Atmel,
- *                2012 Ludovic Desroches <ludovic.desroches@atmel.com>
- *
- * Licensed under GPLv2 or later.
- */
-/dts-v1/;
-/include/ "sama5d3.dtsi"
-/include/ "sama5d3cm.dtsi"
-
-/ {
-	model = "Atmel SAMA5D3-EK";
-	compatible = "atmel,sama5ek", "atmel,sama5d3", "atmel,sama5";
-	atmel,mb-rev = "B";
-
-	chosen {
-		bootargs = "console=ttyS0,115200 mtdparts=atmel_nand:8M(bootstrap/uboot/kernel)ro,-(rootfs) rw rootfstype=ubifs ubi.mtd=1 root=ubi0:rootfs";
-	};
-
-	ahb {
-		apb {
-			mmc0: mmc@f0000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 17 0>;
-				};
-			};
-
-			spi0: spi@f0004000 {
-				status = "okay";
-				m25p80@0 {
-					compatible = "atmel,at25df321a";
-					spi-max-frequency = <50000000>;
-					reg = <0>;
-				};
-			};
-
-			ssc0: ssc@f0008000 {
-				status = "okay";
-			};
-
-			i2c0: i2c@f0014000 {
-				status = "okay";
-
-				wm8904: wm8904@1a {
-					compatible = "wm8904";
-					reg = <0x1a>;
-				};
-			};
-
-			i2c1: i2c@f0018000 {
-				status = "okay";
-
-				qt1070: keyboard@1b {
-					compatible = "qt1070";
-					reg = <0x1b>;
-					interrupt-parent = <&pioE>;
-					interrupts = <31 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_qt1070_irq>;
-				};
-			};
-
-			usart1: serial@f0020000 {
-				pinctrl-0 = <&pinctrl_usart1 &pinctrl_usart1_rts_cts>;
-				status = "okay";
-			};
-
-			lcd_bus@f0030000 {
-				status = "okay";
-
-				lcd@f0030000 {
-					status = "okay";
-				};
-
-				lcdovl1@f0030140 {
-					status = "okay";
-				};
-
-				lcdovl2@f0030240 {
-					status = "okay";
-				};
-
-				lcdheo1@f0030340 {
-					status = "okay";
-				};
-			};
-
-			isi: isi@f0034000 {
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_isi &pinctrl_isi_pck_as_mck_mb_revB &pinctrl_isi_camera_mb_revB>;
-				status = "disabled";
-			};
-
-			mmc1: mmc@f8000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc1_clk_cmd_dat0 &pinctrl_mmc1_dat1_3 &pinctrl_mmc1_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 18 0>;
-				};
-			};
-
-			tsadcc: tsadcc@f8018000 {
-				status = "okay";
-			};
-
-			adc0: adc@f8018000 {
-				status = "disabled";
-			};
-
-			macb1: ethernet@f802c000 {
-				phy-mode = "rmii";
-				status = "okay";
-			};
-
-			pinctrl@fffff200 {
-				board {
-					pinctrl_mmc0_cd: mmc0_cd {
-						atmel,pins =
-							<3 17 0x0 0x5>; /* PD17 GPIO with pullup deglitch */
-					};
-
-					pinctrl_mmc1_cd: mmc1_cd {
-						atmel,pins =
-							<3 18 0x0 0x5>; /* PD18 GPIO with pullup deglitch */
-					};
-
-					pinctrl_pck0_as_audio_mck: pck0_as_audio_mck {
-						atmel,pins =
-							<3 30 0x2 0x0>;	/* PD30 periph B */
-					};
-
-					pinctrl_qt1070_irq: qt1070_irq {
-						atmel,pins =
-							<4 31 0x0 0x5>;	/* PE31 GPIO with pull up deglith */
-					};
-				};
-			};
-
-			dbgu: serial@ffffee00 {
-				status = "okay";
-			};
-
-			watchdog@fffffe40 {
-				status = "okay";
-			};
-		};
-
-		usb0: gadget@00500000 {
-			atmel,vbus-gpio = <&pioD 29 1>;
-			status = "okay";
-		};
-
-		usb1: ohci@00600000 {
-			status = "okay";
-			num-ports = <3>;
-			atmel,vbus-gpio = <&pioD 25 1
-					   &pioD 26 1
-					   &pioD 27 1
-					  >;
-		};
-
-		usb2: ehci@00700000 {
-			status = "okay";
-		};
-	};
-
-	leds {
-		d3 {
-			label = "d3";
-			gpios = <&pioE 24 0>;	/* PE24, conflicts with RTS2 */
-			linux,default-trigger = "heartbeat";
-		};
-	};
-
-	sound {
-		compatible = "atmel,sama5d3ek-wm8904";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_pck0_as_audio_mck>;
-
-		atmel,model = "wm8904 @ SAMA5D3EK";
-		atmel,audio-routing =
-			"MICBIAS", "IN1L",
-			"Left Capture Mux", "MICBIAS";
-
-		atmel,ssc-controller = <&ssc0>;
-		atmel,audio-codec = <&wm8904>;
-	};
-};
diff --git a/arch/arm/boot/dts/sama5d31ek_mb-revB_pda.dts b/arch/arm/boot/dts/sama5d31ek_mb-revB_pda.dts
deleted file mode 100644
index 63dce78..0000000
--- a/arch/arm/boot/dts/sama5d31ek_mb-revB_pda.dts
+++ /dev/null
@@ -1,218 +0,0 @@
-/*
- * sama5d31ek.dts - Device Tree file for SAMA5-EK board
- *
- *  Copyright (C) 2012 Atmel,
- *                2012 Ludovic Desroches <ludovic.desroches@atmel.com>
- *
- * Licensed under GPLv2 or later.
- */
-/dts-v1/;
-/include/ "sama5d3.dtsi"
-/include/ "sama5d3cm.dtsi"
-
-/ {
-	model = "Atmel SAMA5D3-EK";
-	compatible = "atmel,sama5ek", "atmel,sama5d3", "atmel,sama5";
-	atmel,mb-rev = "B";
-
-	chosen {
-		bootargs = "console=ttyS0,115200 mtdparts=atmel_nand:8M(bootstrap/uboot/kernel)ro,-(rootfs) rw rootfstype=ubifs ubi.mtd=1 root=ubi0:rootfs";
-	};
-
-	ahb {
-		apb {
-			mmc0: mmc@f0000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 17 0>;
-				};
-			};
-
-			spi0: spi@f0004000 {
-				status = "okay";
-				m25p80@0 {
-					compatible = "atmel,at25df321a";
-					spi-max-frequency = <50000000>;
-					reg = <0>;
-				};
-			};
-
-			ssc0: ssc@f0008000 {
-				status = "okay";
-			};
-
-			i2c0: i2c@f0014000 {
-				status = "okay";
-
-				wm8904: wm8904@1a {
-					compatible = "wm8904";
-					reg = <0x1a>;
-				};
-			};
-
-			i2c1: i2c@f0018000 {
-				status = "okay";
-
-				qt1070: keyboard@1b {
-					compatible = "qt1070";
-					reg = <0x1b>;
-					interrupt-parent = <&pioE>;
-					interrupts = <30 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_qt1070_irq>;
-				};
-
-				atmel_mxt_ts@4a {
-					compatible = "atmel,atmel_mxt_ts";
-					reg = <0x4a>;
-					interrupt-parent = <&pioE>;
-					interrupts = <31 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_mxt_ts>;
-					x_line = <19>;
-					y_line = <11>;
-					x_size = <1024>;
-					y_size = <1024>;
-					blen = <0x0>;
-					threshold = <0x23>;
-					voltage = <3300000>;
-					orient = <0x00>;
-				};
-			};
-
-			usart1: serial@f0020000 {
-				pinctrl-0 = <&pinctrl_usart1 &pinctrl_usart1_rts_cts>;
-				status = "okay";
-			};
-
-			lcd_bus@f0030000 {
-				status = "okay";
-
-				lcd@f0030000 {
-					status = "okay";
-				};
-
-				lcdovl1@f0030140 {
-					status = "okay";
-				};
-
-				lcdovl2@f0030240 {
-					status = "okay";
-				};
-
-				lcdheo1@f0030340 {
-					status = "okay";
-				};
-			};
-
-			isi: isi@f0034000 {
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_isi &pinctrl_isi_pck_as_mck_mb_revB &pinctrl_isi_camera_mb_revB>;
-				status = "disabled";
-			};
-
-			mmc1: mmc@f8000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc1_clk_cmd_dat0 &pinctrl_mmc1_dat1_3 &pinctrl_mmc1_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 18 0>;
-				};
-			};
-
-			tsadcc: tsadcc@f8018000 {
-				status = "disabled";
-			};
-
-			adc0: adc@f8018000 {
-				status = "disabled";
-			};
-
-			macb1: ethernet@f802c000 {
-				phy-mode = "rmii";
-				status = "okay";
-			};
-
-			pinctrl@fffff200 {
-				board {
-					pinctrl_mmc0_cd: mmc0_cd {
-						atmel,pins =
-							<3 17 0x0 0x5>; /* PD17 GPIO with pullup deglitch */
-					};
-
-					pinctrl_mmc1_cd: mmc1_cd {
-						atmel,pins =
-							<3 18 0x0 0x5>; /* PD18 GPIO with pullup deglitch */
-					};
-
-					pinctrl_pck0_as_audio_mck: pck0_as_audio_mck {
-						atmel,pins =
-							<3 30 0x2 0x0>;	/* PD30 periph B */
-					};
-
-					pinctrl_qt1070_irq: qt1070_irq {
-						atmel,pins =
-							<4 30 0x0 0x5>;	/* PE30 GPIO with pull up deglith */
-					};
-
-					pinctrl_mxt_ts: mxt_irq {
-						atmel,pins =
-							<4 31 0x0 0x5>; /* PE31 gpio with pullup deglitch */
-					};
-				};
-			};
-
-			dbgu: serial@ffffee00 {
-				status = "okay";
-			};
-
-			watchdog@fffffe40 {
-				status = "okay";
-			};
-		};
-
-		usb0: gadget@00500000 {
-			atmel,vbus-gpio = <&pioD 29 1>;
-			status = "okay";
-		};
-
-		usb1: ohci@00600000 {
-			status = "okay";
-			num-ports = <3>;
-			atmel,vbus-gpio = <&pioD 25 1
-					   &pioD 26 1
-					   &pioD 27 1
-					  >;
-		};
-
-		usb2: ehci@00700000 {
-			status = "okay";
-		};
-	};
-
-	leds {
-		d3 {
-			label = "d3";
-			gpios = <&pioE 24 0>;	/* PE24, conflicts with RTS2 */
-			linux,default-trigger = "heartbeat";
-		};
-	};
-
-	sound {
-		compatible = "atmel,sama5d3ek-wm8904";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_pck0_as_audio_mck>;
-
-		atmel,model = "wm8904 @ SAMA5D3EK";
-		atmel,audio-routing =
-			"MICBIAS", "IN1L",
-			"Left Capture Mux", "MICBIAS";
-
-		atmel,ssc-controller = <&ssc0>;
-		atmel,audio-codec = <&wm8904>;
-	};
-};
diff --git a/arch/arm/boot/dts/sama5d33ek_mb-revB.dts b/arch/arm/boot/dts/sama5d33ek_mb-revB.dts
deleted file mode 100644
index d94a13e..0000000
--- a/arch/arm/boot/dts/sama5d33ek_mb-revB.dts
+++ /dev/null
@@ -1,196 +0,0 @@
-/*
- * sama5d33ek.dts - Device Tree file for SAMA5-EK board
- *
- *  Copyright (C) 2012 Atmel,
- *                2012 Ludovic Desroches <ludovic.desroches@atmel.com>
- *
- * Licensed under GPLv2 or later.
- */
-/dts-v1/;
-/include/ "sama5d3.dtsi"
-/include/ "sama5d3cm.dtsi"
-
-/ {
-	model = "Atmel SAMA5D3-EK";
-	compatible = "atmel,sama5ek", "atmel,sama5d3", "atmel,sama5";
-	atmel,mb-rev = "B";
-
-	chosen {
-		bootargs = "console=ttyS0,115200 mtdparts=atmel_nand:8M(bootstrap/uboot/kernel)ro,-(rootfs) rw rootfstype=ubifs ubi.mtd=1 root=ubi0:rootfs";
-	};
-
-	ahb {
-		apb {
-			mmc0: mmc@f0000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 17 0>;
-				};
-			};
-
-			spi0: spi@f0004000 {
-				status = "okay";
-				m25p80@0 {
-					compatible = "atmel,at25df321a";
-					spi-max-frequency = <50000000>;
-					reg = <0>;
-				};
-			};
-
-			ssc0: ssc@f0008000 {
-				status = "okay";
-			};
-
-			i2c0: i2c@f0014000 {
-				status = "okay";
-
-				wm8904: wm8904@1a {
-					compatible = "wm8904";
-					reg = <0x1a>;
-				};
-			};
-
-			i2c1: i2c@f0018000 {
-				status = "okay";
-
-				qt1070: keyboard@1b {
-					compatible = "qt1070";
-					reg = <0x1b>;
-					interrupt-parent = <&pioE>;
-					interrupts = <31 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_qt1070_irq>;
-				};
-			};
-
-			usart1: serial@f0020000 {
-				pinctrl-0 = <&pinctrl_usart1 &pinctrl_usart1_rts_cts>;
-				status = "okay";
-			};
-
-			macb0: ethernet@f0028000 {
-				phy-mode = "rgmii";
-				status = "okay";
-			};
-
-			lcd_bus@f0030000 {
-				status = "okay";
-
-				lcd@f0030000 {
-					status = "okay";
-				};
-
-				lcdovl1@f0030140 {
-					status = "okay";
-				};
-
-				lcdovl2@f0030240 {
-					status = "okay";
-				};
-
-				lcdheo1@f0030340 {
-					status = "okay";
-				};
-			};
-
-			isi: isi@f0034000 {
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_isi &pinctrl_isi_pck_as_mck_mb_revB &pinctrl_isi_camera_mb_revB>;
-				status = "disabled";
-			};
-
-			mmc1: mmc@f8000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc1_clk_cmd_dat0 &pinctrl_mmc1_dat1_3 &pinctrl_mmc1_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 18 0>;
-				};
-			};
-
-			tsadcc: tsadcc@f8018000 {
-				status = "okay";
-			};
-
-			adc0: adc@f8018000 {
-				status = "disabled";
-			};
-
-			pinctrl@fffff200 {
-				board {
-					pinctrl_mmc0_cd: mmc0_cd {
-						atmel,pins =
-							<3 17 0x0 0x5>; /* PD17 GPIO with pullup deglitch */
-					};
-
-					pinctrl_mmc1_cd: mmc1_cd {
-						atmel,pins =
-							<3 18 0x0 0x5>; /* PD18 GPIO with pullup deglitch */
-					};
-
-					pinctrl_pck0_as_audio_mck: pck0_as_audio_mck {
-						atmel,pins =
-							<3 30 0x2 0x0>;	/* PD30 periph B */
-					};
-
-					pinctrl_qt1070_irq: qt1070_irq {
-						atmel,pins =
-							<4 31 0x0 0x5>;	/* PE31 GPIO with pull up deglith */
-					};
-				};
-			};
-
-			dbgu: serial@ffffee00 {
-				status = "okay";
-			};
-
-			watchdog@fffffe40 {
-				status = "okay";
-			};
-		};
-
-		usb0: gadget@00500000 {
-			atmel,vbus-gpio = <&pioD 29 1>;
-			status = "okay";
-		};
-
-		usb1: ohci@00600000 {
-			status = "okay";
-			num-ports = <3>;
-			atmel,vbus-gpio = <&pioD 25 1
-					   &pioD 26 1
-					   &pioD 27 1
-					  >;
-		};
-
-		usb2: ehci@00700000 {
-			status = "okay";
-		};
-	};
-
-	leds {
-		d3 {
-			label = "d3";
-			gpios = <&pioE 24 0>;	/* PE24, conflicts with RTS2 */
-			linux,default-trigger = "heartbeat";
-		};
-	};
-
-	sound {
-		compatible = "atmel,sama5d3ek-wm8904";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_pck0_as_audio_mck>;
-
-		atmel,model = "wm8904 @ SAMA5D3EK";
-		atmel,audio-routing =
-			"MICBIAS", "IN1L",
-			"Left Capture Mux", "MICBIAS";
-
-		atmel,ssc-controller = <&ssc0>;
-		atmel,audio-codec = <&wm8904>;
-	};
-};
diff --git a/arch/arm/boot/dts/sama5d33ek_mb-revB_pda.dts b/arch/arm/boot/dts/sama5d33ek_mb-revB_pda.dts
deleted file mode 100644
index 7803f86..0000000
--- a/arch/arm/boot/dts/sama5d33ek_mb-revB_pda.dts
+++ /dev/null
@@ -1,218 +0,0 @@
-/*
- * sama5d33ek.dts - Device Tree file for SAMA5-EK board
- *
- *  Copyright (C) 2012 Atmel,
- *                2012 Ludovic Desroches <ludovic.desroches@atmel.com>
- *
- * Licensed under GPLv2 or later.
- */
-/dts-v1/;
-/include/ "sama5d3.dtsi"
-/include/ "sama5d3cm.dtsi"
-
-/ {
-	model = "Atmel SAMA5D3-EK";
-	compatible = "atmel,sama5ek", "atmel,sama5d3", "atmel,sama5";
-	atmel,mb-rev = "B";
-
-	chosen {
-		bootargs = "console=ttyS0,115200 mtdparts=atmel_nand:8M(bootstrap/uboot/kernel)ro,-(rootfs) rw rootfstype=ubifs ubi.mtd=1 root=ubi0:rootfs";
-	};
-
-	ahb {
-		apb {
-			mmc0: mmc@f0000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 17 0>;
-				};
-			};
-
-			spi0: spi@f0004000 {
-				status = "okay";
-				m25p80@0 {
-					compatible = "atmel,at25df321a";
-					spi-max-frequency = <50000000>;
-					reg = <0>;
-				};
-			};
-
-			ssc0: ssc@f0008000 {
-				status = "okay";
-			};
-
-			i2c0: i2c@f0014000 {
-				status = "okay";
-
-				wm8904: wm8904@1a {
-					compatible = "wm8904";
-					reg = <0x1a>;
-				};
-			};
-
-			i2c1: i2c@f0018000 {
-				status = "okay";
-
-				qt1070: keyboard@1b {
-					compatible = "qt1070";
-					reg = <0x1b>;
-					interrupt-parent = <&pioE>;
-					interrupts = <30 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_qt1070_irq>;
-				};
-
-				atmel_mxt_ts@4a {
-					compatible = "atmel,atmel_mxt_ts";
-					reg = <0x4a>;
-					interrupt-parent = <&pioE>;
-					interrupts = <31 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_mxt_ts>;
-					x_line = <19>;
-					y_line = <11>;
-					x_size = <1024>;
-					y_size = <1024>;
-					blen = <0x0>;
-					threshold = <0x23>;
-					voltage = <3300000>;
-					orient = <0x00>;
-				};
-			};
-
-			usart1: serial@f0020000 {
-				pinctrl-0 = <&pinctrl_usart1 &pinctrl_usart1_rts_cts>;
-				status = "okay";
-			};
-
-			macb0: ethernet@f0028000 {
-				phy-mode = "rgmii";
-				status = "okay";
-			};
-
-			lcd_bus@f0030000 {
-				status = "okay";
-
-				lcd@f0030000 {
-					status = "okay";
-				};
-
-				lcdovl1@f0030140 {
-					status = "okay";
-				};
-
-				lcdovl2@f0030240 {
-					status = "okay";
-				};
-
-				lcdheo1@f0030340 {
-					status = "okay";
-				};
-			};
-
-			isi: isi@f0034000 {
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_isi &pinctrl_isi_pck_as_mck_mb_revB &pinctrl_isi_camera_mb_revB>;
-				status = "disabled";
-			};
-
-			mmc1: mmc@f8000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc1_clk_cmd_dat0 &pinctrl_mmc1_dat1_3 &pinctrl_mmc1_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 18 0>;
-				};
-			};
-
-			tsadcc: tsadcc@f8018000 {
-				status = "disabled";
-			};
-
-			adc0: adc@f8018000 {
-				status = "disabled";
-			};
-
-			pinctrl@fffff200 {
-				board {
-					pinctrl_mmc0_cd: mmc0_cd {
-						atmel,pins =
-							<3 17 0x0 0x5>; /* PD17 GPIO with pullup deglitch */
-					};
-
-					pinctrl_mmc1_cd: mmc1_cd {
-						atmel,pins =
-							<3 18 0x0 0x5>; /* PD18 GPIO with pullup deglitch */
-					};
-
-					pinctrl_pck0_as_audio_mck: pck0_as_audio_mck {
-						atmel,pins =
-							<3 30 0x2 0x0>;	/* PD30 periph B */
-					};
-
-					pinctrl_qt1070_irq: qt1070_irq {
-						atmel,pins =
-							<4 30 0x0 0x5>;	/* PE30 GPIO with pull up deglith */
-					};
-
-					pinctrl_mxt_ts: mxt_irq {
-						atmel,pins =
-							<4 31 0x0 0x5>; /* PE31 gpio with pullup deglitch */
-					};
-				};
-			};
-
-			dbgu: serial@ffffee00 {
-				status = "okay";
-			};
-
-			watchdog@fffffe40 {
-				status = "okay";
-			};
-		};
-
-		usb0: gadget@00500000 {
-			atmel,vbus-gpio = <&pioD 29 1>;
-			status = "okay";
-		};
-
-		usb1: ohci@00600000 {
-			status = "okay";
-			num-ports = <3>;
-			atmel,vbus-gpio = <&pioD 25 1
-					   &pioD 26 1
-					   &pioD 27 1
-					  >;
-		};
-
-		usb2: ehci@00700000 {
-			status = "okay";
-		};
-	};
-
-	leds {
-		d3 {
-			label = "d3";
-			gpios = <&pioE 24 0>;	/* PE24, conflicts with RTS2 */
-			linux,default-trigger = "heartbeat";
-		};
-	};
-
-	sound {
-		compatible = "atmel,sama5d3ek-wm8904";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_pck0_as_audio_mck>;
-
-		atmel,model = "wm8904 @ SAMA5D3EK";
-		atmel,audio-routing =
-			"MICBIAS", "IN1L",
-			"Left Capture Mux", "MICBIAS";
-
-		atmel,ssc-controller = <&ssc0>;
-		atmel,audio-codec = <&wm8904>;
-	};
-};
diff --git a/arch/arm/boot/dts/sama5d34ek_mb-revB.dts b/arch/arm/boot/dts/sama5d34ek_mb-revB.dts
deleted file mode 100644
index 28a84a2..0000000
--- a/arch/arm/boot/dts/sama5d34ek_mb-revB.dts
+++ /dev/null
@@ -1,206 +0,0 @@
-/*
- * sama5d34ek.dts - Device Tree file for SAMA5-EK board
- *
- *  Copyright (C) 2012 Atmel,
- *                2012 Ludovic Desroches <ludovic.desroches@atmel.com>
- *
- * Licensed under GPLv2 or later.
- */
-/dts-v1/;
-/include/ "sama5d3.dtsi"
-/include/ "sama5d3cm.dtsi"
-
-/ {
-	model = "Atmel SAMA5D3-EK";
-	compatible = "atmel,sama5ek", "atmel,sama5d3", "atmel,sama5";
-	atmel,mb-rev = "B";
-
-	chosen {
-		bootargs = "console=ttyS0,115200 mtdparts=atmel_nand:8M(bootstrap/uboot/kernel)ro,-(rootfs) rw rootfstype=ubifs ubi.mtd=1 root=ubi0:rootfs";
-	};
-
-	ahb {
-		apb {
-			mmc0: mmc@f0000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 17 0>;
-				};
-			};
-
-			spi0: spi@f0004000 {
-				status = "okay";
-				m25p80@0 {
-					compatible = "atmel,at25df321a";
-					spi-max-frequency = <50000000>;
-					reg = <0>;
-				};
-			};
-
-			ssc0: ssc@f0008000 {
-				status = "okay";
-			};
-
-			can0: can@f000c000 {
-				status = "okay";
-			};
-
-			i2c0: i2c@f0014000 {
-				status = "okay";
-
-				wm8904: wm8904@1a {
-					compatible = "wm8904";
-					reg = <0x1a>;
-				};
-			};
-
-			i2c1: i2c@f0018000 {
-				status = "okay";
-
-				24c256@50 {
-					compatible = "24c256";
-					reg = <0x50>;
-					pagesize = <64>;
-				};
-
-				qt1070: keyboard@1b {
-					compatible = "qt1070";
-					reg = <0x1b>;
-					interrupt-parent = <&pioE>;
-					interrupts = <31 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_qt1070_irq>;
-				};
-			};
-
-			usart1: serial@f0020000 {
-				pinctrl-0 = <&pinctrl_usart1 &pinctrl_usart1_rts_cts>;
-				status = "okay";
-			};
-
-			macb0: ethernet@f0028000 {
-				phy-mode = "rgmii";
-				status = "okay";
-			};
-
-			lcd_bus@f0030000 {
-				status = "okay";
-
-				lcd@f0030000 {
-					status = "okay";
-				};
-
-				lcdovl1@f0030140 {
-					status = "okay";
-				};
-
-				lcdovl2@f0030240 {
-					status = "okay";
-				};
-
-				lcdheo1@f0030340 {
-					status = "okay";
-				};
-			};
-
-			isi: isi@f0034000 {
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_isi &pinctrl_isi_pck_as_mck_mb_revB &pinctrl_isi_camera_mb_revB>;
-				status = "disabled";
-			};
-
-			mmc1: mmc@f8000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc1_clk_cmd_dat0 &pinctrl_mmc1_dat1_3 &pinctrl_mmc1_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 18 0>;
-				};
-			};
-
-			tsadcc: tsadcc@f8018000 {
-				status = "okay";
-			};
-
-			adc0: adc@f8018000 {
-				status = "disabled";
-			};
-
-			pinctrl@fffff200 {
-				board {
-					pinctrl_mmc0_cd: mmc0_cd {
-						atmel,pins =
-							<3 17 0x0 0x5>;	/* PD17 GPIO with pullup deglitch */
-					};
-
-					pinctrl_mmc1_cd: mmc1_cd {
-						atmel,pins =
-							<3 18 0x0 0x5>;	/* PD18 GPIO with pullup deglitch */
-					};
-
-					pinctrl_pck0_as_audio_mck: pck0_as_audio_mck {
-						atmel,pins =
-							<3 30 0x2 0x0>;	/* PD30 periph B */
-					};
-
-					pinctrl_qt1070_irq: qt1070_irq {
-						atmel,pins =
-							<4 31 0x0 0x5>;	/* PE31 GPIO with pull up deglith */
-					};
-				};
-			};
-
-			dbgu: serial@ffffee00 {
-				status = "okay";
-			};
-
-			watchdog@fffffe40 {
-				status = "okay";
-			};
-		};
-
-		usb0: gadget@00500000 {
-			atmel,vbus-gpio = <&pioD 29 1>;
-			status = "okay";
-		};
-
-		usb1: ohci@00600000 {
-			status = "okay";
-			num-ports = <3>;
-			atmel,vbus-gpio = <&pioD 25 1
-					   &pioD 26 1
-					   &pioD 27 1
-					  >;
-		};
-
-		usb2: ehci@00700000 {
-			status = "okay";
-		};
-	};
-
-	leds {
-		d3 {
-			label = "d3";
-			gpios = <&pioE 24 0>;	/* PE24, conflicts with RTS2 */
-			linux,default-trigger = "heartbeat";
-		};
-	};
-
-	sound {
-		compatible = "atmel,sama5d3ek-wm8904";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_pck0_as_audio_mck>;
-
-		atmel,model = "wm8904 @ SAMA5D3EK";
-		atmel,audio-routing =
-			"MICBIAS", "IN1L",
-			"Left Capture Mux", "MICBIAS";
-
-		atmel,ssc-controller = <&ssc0>;
-		atmel,audio-codec = <&wm8904>;
-	};
-};
diff --git a/arch/arm/boot/dts/sama5d34ek_mb-revB_pda.dts b/arch/arm/boot/dts/sama5d34ek_mb-revB_pda.dts
deleted file mode 100644
index 7608ee9..0000000
--- a/arch/arm/boot/dts/sama5d34ek_mb-revB_pda.dts
+++ /dev/null
@@ -1,228 +0,0 @@
-/*
- * sama5d34ek.dts - Device Tree file for SAMA5-EK board
- *
- *  Copyright (C) 2012 Atmel,
- *                2012 Ludovic Desroches <ludovic.desroches@atmel.com>
- *
- * Licensed under GPLv2 or later.
- */
-/dts-v1/;
-/include/ "sama5d3.dtsi"
-/include/ "sama5d3cm.dtsi"
-
-/ {
-	model = "Atmel SAMA5D3-EK";
-	compatible = "atmel,sama5ek", "atmel,sama5d3", "atmel,sama5";
-	atmel,mb-rev = "B";
-
-	chosen {
-		bootargs = "console=ttyS0,115200 mtdparts=atmel_nand:8M(bootstrap/uboot/kernel)ro,-(rootfs) rw rootfstype=ubifs ubi.mtd=1 root=ubi0:rootfs";
-	};
-
-	ahb {
-		apb {
-			mmc0: mmc@f0000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 17 0>;
-				};
-			};
-
-			spi0: spi@f0004000 {
-				status = "okay";
-				m25p80@0 {
-					compatible = "atmel,at25df321a";
-					spi-max-frequency = <50000000>;
-					reg = <0>;
-				};
-			};
-
-			ssc0: ssc@f0008000 {
-				status = "okay";
-			};
-
-			can0: can@f000c000 {
-				status = "okay";
-			};
-
-			i2c0: i2c@f0014000 {
-				status = "okay";
-
-				wm8904: wm8904@1a {
-					compatible = "wm8904";
-					reg = <0x1a>;
-				};
-			};
-
-			i2c1: i2c@f0018000 {
-				status = "okay";
-
-				24c256@50 {
-					compatible = "24c256";
-					reg = <0x50>;
-					pagesize = <64>;
-				};
-
-				qt1070: keyboard@1b {
-					compatible = "qt1070";
-					reg = <0x1b>;
-					interrupt-parent = <&pioE>;
-					interrupts = <30 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_qt1070_irq>;
-				};
-
-				atmel_mxt_ts@4a {
-					compatible = "atmel,atmel_mxt_ts";
-					reg = <0x4a>;
-					interrupt-parent = <&pioE>;
-					interrupts = <31 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_mxt_ts>;
-					x_line = <19>;
-					y_line = <11>;
-					x_size = <1024>;
-					y_size = <1024>;
-					blen = <0x0>;
-					threshold = <0x23>;
-					voltage = <3300000>;
-					orient = <0x00>;
-				};
-			};
-
-			usart1: serial@f0020000 {
-				pinctrl-0 = <&pinctrl_usart1 &pinctrl_usart1_rts_cts>;
-				status = "okay";
-			};
-
-			macb0: ethernet@f0028000 {
-				phy-mode = "rgmii";
-				status = "okay";
-			};
-
-			lcd_bus@f0030000 {
-				status = "okay";
-
-				lcd@f0030000 {
-					status = "okay";
-				};
-
-				lcdovl1@f0030140 {
-					status = "okay";
-				};
-
-				lcdovl2@f0030240 {
-					status = "okay";
-				};
-
-				lcdheo1@f0030340 {
-					status = "okay";
-				};
-			};
-
-			isi: isi@f0034000 {
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_isi &pinctrl_isi_pck_as_mck_mb_revB &pinctrl_isi_camera_mb_revB>;
-				status = "disabled";
-			};
-
-			mmc1: mmc@f8000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc1_clk_cmd_dat0 &pinctrl_mmc1_dat1_3 &pinctrl_mmc1_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 18 0>;
-				};
-			};
-
-			tsadcc: tsadcc@f8018000 {
-				status = "disabled";
-			};
-
-			adc0: adc@f8018000 {
-				status = "disabled";
-			};
-
-			pinctrl@fffff200 {
-				board {
-					pinctrl_mmc0_cd: mmc0_cd {
-						atmel,pins =
-							<3 17 0x0 0x5>;	/* PD17 GPIO with pullup deglitch */
-					};
-
-					pinctrl_mmc1_cd: mmc1_cd {
-						atmel,pins =
-							<3 18 0x0 0x5>;	/* PD18 GPIO with pullup deglitch */
-					};
-
-					pinctrl_pck0_as_audio_mck: pck0_as_audio_mck {
-						atmel,pins =
-							<3 30 0x2 0x0>;	/* PD30 periph B */
-					};
-
-					pinctrl_qt1070_irq: qt1070_irq {
-						atmel,pins =
-							<4 30 0x0 0x5>;	/* PE30 GPIO with pull up deglith */
-					};
-
-					pinctrl_mxt_ts: mxt_irq {
-						atmel,pins =
-							<4 31 0x0 0x5>; /* PE31 gpio with pullup deglitch */
-					};
-				};
-			};
-
-			dbgu: serial@ffffee00 {
-				status = "okay";
-			};
-
-			watchdog@fffffe40 {
-				status = "okay";
-			};
-		};
-
-		usb0: gadget@00500000 {
-			atmel,vbus-gpio = <&pioD 29 1>;
-			status = "okay";
-		};
-
-		usb1: ohci@00600000 {
-			status = "okay";
-			num-ports = <3>;
-			atmel,vbus-gpio = <&pioD 25 1
-					   &pioD 26 1
-					   &pioD 27 1
-					  >;
-		};
-
-		usb2: ehci@00700000 {
-			status = "okay";
-		};
-	};
-
-	leds {
-		d3 {
-			label = "d3";
-			gpios = <&pioE 24 0>;	/* PE24, conflicts with RTS2 */
-			linux,default-trigger = "heartbeat";
-		};
-	};
-
-	sound {
-		compatible = "atmel,sama5d3ek-wm8904";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_pck0_as_audio_mck>;
-
-		atmel,model = "wm8904 @ SAMA5D3EK";
-		atmel,audio-routing =
-			"MICBIAS", "IN1L",
-			"Left Capture Mux", "MICBIAS";
-
-		atmel,ssc-controller = <&ssc0>;
-		atmel,audio-codec = <&wm8904>;
-	};
-};
diff --git a/arch/arm/boot/dts/sama5d35ek_mb-revB.dts b/arch/arm/boot/dts/sama5d35ek_mb-revB.dts
deleted file mode 100644
index 38f54aa..0000000
--- a/arch/arm/boot/dts/sama5d35ek_mb-revB.dts
+++ /dev/null
@@ -1,199 +0,0 @@
-/*
- * sama5d35ek.dts - Device Tree file for SAMA5-EK board
- *
- *  Copyright (C) 2012 Atmel,
- *                2012 Ludovic Desroches <ludovic.desroches@atmel.com>
- *
- * Licensed under GPLv2 or later.
- */
-/dts-v1/;
-/include/ "sama5d3.dtsi"
-/include/ "sama5d3cm.dtsi"
-
-/ {
-	model = "Atmel SAMA5D3-EK";
-	compatible = "atmel,sama5ek", "atmel,sama5d3", "atmel,sama5";
-	atmel,mb-rev = "B";
-
-	chosen {
-		bootargs = "console=ttyS0,115200 mtdparts=atmel_nand:8M(bootstrap/uboot/kernel)ro,-(rootfs) rw rootfstype=ubifs ubi.mtd=1 root=ubi0:rootfs";
-	};
-
-	ahb {
-		apb {
-			mmc0: mmc@f0000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 17 0>;
-				};
-			};
-
-			spi0: spi@f0004000 {
-				status = "okay";
-				m25p80@0 {
-					compatible = "atmel,at25df321a";
-					spi-max-frequency = <50000000>;
-					reg = <0>;
-				};
-			};
-
-			ssc0: ssc@f0008000 {
-				status = "disabled";
-			};
-
-			can0: can@f000c000 {
-				status = "okay";
-			};
-
-			/*
-			 * i2c0 conflicts with ISI:
-			 * disable it to allow the use of ISI
-			 * can not enable audio when i2c0 disabled
-			 */
-			i2c0: i2c@f0014000 {
-				status = "disabled";
-
-				wm8904: wm8904@1a {
-					compatible = "wm8904";
-					reg = <0x1a>;
-				};
-			};
-
-			i2c1: i2c@f0018000 {
-				status = "okay";
-
-				/* No LCD support
-				 * not sure enable qt1070 make sense
-				 */
-				qt1070: keyboard@1b {
-					compatible = "qt1070";
-					reg = <0x1b>;
-					interrupt-parent = <&pioE>;
-					interrupts = <31 0x0>;
-					pinctrl-names = "default";
-					pinctrl-0 = <&pinctrl_qt1070_irq>;
-				};
-			};
-
-			usart1: serial@f0020000 {
-				pinctrl-0 = <&pinctrl_usart1 &pinctrl_usart1_rts_cts>;
-				status = "okay";
-			};
-
-			macb0: ethernet@f0028000 {
-				phy-mode = "rgmii";
-				status = "okay";
-			};
-
-			isi: isi@f0034000 {
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_isi &pinctrl_isi_pck_as_mck_mb_revB &pinctrl_isi_camera_mb_revB>;
-				status = "okay";
-			};
-
-			mmc1: mmc@f8000000 {
-				status = "okay";
-				pinctrl-0 = <&pinctrl_mmc1_clk_cmd_dat0 &pinctrl_mmc1_dat1_3 &pinctrl_mmc1_cd>;
-				slot@0 {
-					reg = <0>;
-					bus-width = <4>;
-					cd-gpios = <&pioD 18 0>;
-				};
-			};
-
-			adc0: adc@f8018000 {
-				status = "okay";
-				pinctrl-0 = <
-					&pinctrl_adc0_adtrg
-					&pinctrl_adc0_ad0
-					&pinctrl_adc0_ad1
-					&pinctrl_adc0_ad2
-					&pinctrl_adc0_ad3
-					&pinctrl_adc0_ad4
-					>;
-			};
-
-			macb1: ethernet@f802c000 {
-				phy-mode = "rmii";
-				status = "okay";
-			};
-
-			pinctrl@fffff200 {
-				board {
-					pinctrl_mmc0_cd: mmc0_cd {
-						atmel,pins =
-							<3 17 0x0 0x5>; /* PD17 GPIO with pullup deglitch */
-					};
-
-					pinctrl_mmc1_cd: mmc1_cd {
-						atmel,pins =
-							<3 18 0x0 0x5>; /* PD18 GPIO with pullup deglitch */
-					};
-
-					pinctrl_pck0_as_audio_mck: pck0_as_audio_mck {
-						atmel,pins =
-							<3 30 0x2 0x0>;	/* PD30 periph B */
-					};
-
-					pinctrl_qt1070_irq: qt1070_irq {
-						atmel,pins =
-							<4 31 0x0 0x5>;	/* PE31 GPIO with pull up deglith */
-					};
-				};
-			};
-
-			dbgu: serial@ffffee00 {
-				status = "okay";
-			};
-
-			watchdog@fffffe40 {
-				status = "okay";
-			};
-		};
-
-		usb0: gadget@00500000 {
-			atmel,vbus-gpio = <&pioD 29 1>;
-			status = "okay";
-		};
-
-		usb1: ohci@00600000 {
-			status = "okay";
-			num-ports = <3>;
-			atmel,vbus-gpio = <&pioD 25 1
-					   &pioD 26 1
-					   &pioD 27 1
-					  >;
-		};
-
-		usb2: ehci@00700000 {
-			status = "okay";
-		};
-	};
-
-	leds {
-		d3 {
-			label = "d3";
-			gpios = <&pioE 24 0>;	/* PE24, conflicts with RTS2 */
-			linux,default-trigger = "heartbeat";
-		};
-	};
-
-	sound {
-		compatible = "atmel,sama5d3ek-wm8904";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_pck0_as_audio_mck>;
-
-		atmel,model = "wm8904 @ SAMA5D3EK";
-		atmel,audio-routing =
-			"MICBIAS", "IN1L",
-			"Left Capture Mux", "MICBIAS";
-
-		atmel,ssc-controller = <&ssc0>;
-		atmel,audio-codec = <&wm8904>;
-
-		status = "disabled";
-	};
-};
-- 
1.7.10.4

